sw/device/lib/runtime/check.h

```diff
@@ -23,12 +23,17 @@
  * @param ... arguments to a LOG_* macro, which are evaluated if the check
  * fails.
  */
-#define CHECK(condition, ...)                \
-  do {                                       \
-    if (!(condition)) {                      \
-      LOG_ERROR("CHECK-fail: " __VA_ARGS__); \
-      test_status_set(kTestStatusFailed);    \
-    }                                        \
+#define CHECK(condition, ...)                 \
+  do {                                        \
+    if (!(condition)) {                       \
+      LOG_ERROR("CHECK-fail: " __VA_ARGS__);  \
+      /* Currently, this macro will call into \
+         the test failure code, which logs    \
+         "FAIL" and aborts. In the future,    \
+         we will try to condition on whether  \
+         or not this is a test.*/             \
+      set_test_status(kTestStatusFailed);     \
```

Comment:
This get called `test_status_set` in the commit, didn't it?

Reply:
Oops... fixed.